packages/app/src/sandbox/react-error-overlay/overlay.js

```diff
@@ -163,7 +163,7 @@ function disposeCurrentView(force: boolean) {
 }
 
 function unmount(force: boolean = false) {
```

Comment:
Would it make more sense to set `force` by default to `true`? We have some logic behind forcing (https://github.com/CompuIves/codesandbox-client/blob/master/packages/app/src/sandbox/compile.js#L309). Other than that this looks good!